joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -67,6 +67,9 @@ controller_interface::CallbackReturn JointTrajectoryController::on_init()
       "allow_integration_in_goal_trajectories", allow_integration_in_goal_trajectories_);
     state_publish_rate_ = auto_declare<double>("state_publish_rate", 50.0);
     action_monitor_rate_ = auto_declare<double>("action_monitor_rate", 20.0);
+
+    std::string interpolation_string = auto_declare<std::string>("interpolation_method", "splines");
```

Comment:
Since these strings are managed by the enum method. it would make sense to define those there so that everything is in one place. Now we have here string definition for parameters in one file and checking the same strings in other file. You could do 2 things, add const expr for each string and also add the full list of those so that all can be simple accessed.

Reply:
I don't completely understand. Shall I do something like this?

https://github.com/ros-planning/moveit2/blob/33b075d15f8d9ec64ca8fea98cf1573b7c581c47/moveit_ros/moveit_servo/include/moveit_servo/status_codes.h#L46

Reply:
Yes, please. So, the strings should be somewhere defined as constants to avoid unintentional error is multiple instances of those get misaligned.

Reply:
OK, done.